Sources/Nakama/SocketClient.swift

```diff
@@ -151,9 +151,9 @@ public protocol SocketClient {
 
      - Parameter target: The target channel to join.
      - Parameter type: The type of channel to join.
-     - Returns: A EventLoopFuture which resolves to a Channel response.
+     - Returns: A nakama realtime channel.
```

Comment:
Small comment -- we should capitalize Nakama

Reply:
@lugehorsam Will update Nakama capitalization in new commit now so we can merge and move to next stage: storage & RPC calls.